bazel

```diff
@@ -55,6 +55,7 @@ processor_name="$(uname -m)"
 bazel_installer_platform="${kernel_name}-${processor_name}"
 case "$bazel_installer_platform" in
   linux-x86_64) ;;
+  linux-arm64) ;;
```

Comment:
Why do we not need to add a checksum to the bazel installer at the end of the `WORKSPACE` file like we do for all the other installers?

https://github.com/sorbet/sorbet/blob/52da7a7809556280402c918409cf2de486db9d5c/WORKSPACE#L62-L68